packages/js/src/components/AddonInstallationSuccessful.js

```diff
@@ -0,0 +1,35 @@
+import { Alert } from "@yoast/components";
+import { __, sprintf } from "@wordpress/i18n";
+import interpolateComponents from "interpolate-components";
+
+/**
+ * Plugin installation successful alert.
+ *
+ * @returns {React.Element} The plugin installation successful alert.
+ */
+const AddonInstallationSuccessful = () => {
+	let text = sprintf(
+		/* translators: %1$s expands to Yoast, %2$s to a 'link' open tag, %2$s to a 'link' close tag. */
+		__( "Installation successful! We hope you'll enjoy %1$s SEO Premium. " +
+			"You can get started by running through %2$sour configuration wizard.%3$s", "wordpress-seo" ),
+		"Yoast",
+		"{{a}}",
+		"{{/a}}"
+	);
+
+	text = interpolateComponents( {
+		mixedString: text,
+		components: {
+			// eslint-disable-next-line jsx-a11y/anchor-has-content
+			a: <a href="/wp-admin/admin.php?page=wpseo_configurator" />,
```

Comment:
This assumes that the admin is placed on the 'default' location, which will work for most cases, but it's pretty easy to make sure we provide the link via the localization method and use the `admin_url` method in PHP to pass it to this component as prop.

_We also pass localized properties to components in the installation modal JavaScript._